docs/pages/blog/2020-q1-update.md

```diff
@@ -0,0 +1,110 @@
+---
+description: Here are the most significant improvements in August.
+---
+
+# Q1 2020 Update
+
+Olivier Tassinari. April 14, 2020.
+
+Welcome to the new format of our mission update. We are moving from monthly to quarterly updates.
+
+## Product
+
+Over the last 3 months, we have focused on making patch releases.
+We have done eleven so far. We have optimized for stability.
+
+Here are the most significant improvements since the beginning of the year:
+
+- 🔍 We have polished the [Autocomplete](https://material-ui.com/components/autocomplete/) component (Combo box). We have closed more than [270](https://github.com/mui-org/material-ui/labels/lab%3A%20Autocomplete) issues and pull requests so far. We will promote the component to the core in the next major (v5).
+
+  ![autocomplete](/static/blog/2020-q1-update/autocomplete.gif)
+
+  <p class="blog-description">useAutocomplete <a href="https://material-ui.com/components/autocomplete/#useautocomplete">hook</a> example, 4.5 kB gzipped.</p>
+
+  If you wish to make your first contribution to open source, the component has a couple of ["good first issue"](https://github.com/mui-org/material-ui/labels/lab%3A%20Autocomplete) that can be taken.
+  If you have TypeScript expertise, the component could [benefit from it](https://github.com/mui-org/material-ui/issues?q=is%3Aopen+label%3A%22lab%3A+Autocomplete%22+label%3Atypescript).
+
+- 📆 We have made a first [alpha release](https://next.material-ui-pickers.dev/demo/daterangepicker) of the date range picker. It's not ready to be used in production but you can start playing. We might release some of the new features of the date picker under the upcoming Enterprise offering.
+
+  ![date picker](/static/blog/2020-q1-update/date-picker.png)
+
+- ⚠️ We have introduced a new [Alert](https://material-ui.com/components/alert/) component in the lab. While this component isn't mentioned in the Material Design specification, it's a common and well-established pattern. For instance, it can be interested when [combined](https://material-ui.com/components/snackbars/#notistack) with the Snackbar.
```

Comment:
Grammatically interested should probably be interesting. But I'm not sure interesting is the right description. 

Maybe something like: For instance, it can be combined with the Snackbar to display a Toast.